ignite/internal/tools/gen-cli-docs/main.go

```diff
@@ -71,13 +71,20 @@ func run(outPath string) error {
 		return err
 	}
 	defer cleanUp()
+	cmd.Flags().String(outFlag, ".", ".md file path to place Ignite CLI docs inside")
+	cmd.Flags().MarkHidden(outFlag)
```

Comment:
why hidden?

Reply:
Otherwise it pops up in the generated cli docs: https://github.com/ignite/cli/pull/4084/commits/25da0651f637c86317b495bd658b44490cf240bd